tasks/fullStack/app/models/shop.rb

```diff
@@ -1,4 +1,27 @@
 class Shop < ApplicationRecord
   has_many :products
   has_many :reviews, through: :products
+  has_many :monthly_avg_ratings
+
+  def ratings_change(months)
+    today = Date.today
+    x_months_ago = today - months.months
+
+    avg_ratings = monthly_avg_ratings
+      .where(start_date: x_months_ago.end_of_month..today.end_of_month)
+
+    sorted_ratings = avg_ratings.sort_by { |rating| rating.start_date }
```

Comment:
Would sorting via DB be better, (run faster and simpler to code), than sorting in Ruby `sort_by`?

Reply:
I agree, this line is redundant.

Reply:
It's not just redundant I think. Because if we remove this line, the query/result may not be correct since the order of the `avg_ratings` may not always be correct.

Reply:
I mean, we can remove this line of code because we can sort `avg_ratings` through the database.